internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -153,20 +153,20 @@ func logConfigOpts(lc *manifest.Logging) *template.LogConfigOpts {
 
 // convertStorageOpts converts a manifest Storage field into template data structures which can be used
 // to execute CFN templates
-func convertStorageOpts(in *manifest.Storage) (*template.StorageOpts, error) {
+func convertStorageOpts(in *manifest.Storage) (template.StorageOpts, error) {
```

Comment:
Do we need to make similar changes to the other convert functions?

Reply:
`convertHTTPHealthCheck` does not need it; the template execution requires that the `HTTPHealthCheckOpts` struct always exists

`convertAutoscaling` already returns a pointer and that is accounted for in the template. 

Same for `logConfigOpts`.